compiler/hash-codegen/src/symbols/mangle.rs

```diff
@@ -68,9 +68,7 @@ pub fn compute_symbol_name(instance_id: InstanceId) -> String {
     let m = &mut Mangler { out: String::new() };
 
     instance_id.map(|instance| {
-        if !instance.attributes.contains(IDENTS.no_mangle)
-            && !instance.attributes.contains(IDENTS.foreign)
-        {
+        if !instance.has_attr(IDENTS.no_mangle) && !instance.has_attr(IDENTS.foreign) {
```

Comment:
Would be cool if the attr map exposes the IDs of the defined attributes so they don't have to be looked up by name